### PR TITLE
Working

### PR DESCRIPTION
**Pull request by PhoebeBC**

### CODE REVIEW

File: ExperiencePage/index.html [hunk 1]
```diff
@@ -0,0 +1,270 @@
+<!--
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Phoebe Badcock Experience</title>
+    <link rel="stylesheet" href="styles.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (MightyWizard) [inline ExperiencePage/index.html]: Download this and have it as a local style sheet

File: ExperiencePage/index.html [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+	<link rel="stylesheet" media="screen and (min-height: 800px)" href="monitor.css">
+	<link rel="stylesheet" media="screen and (max-width: 450px)" href="mobile.css">
+</head>
+<body>
+  &lt;!&ndash; START NAV &ndash;&gt;
+    <nav id="nav">
+		<div id="name">
+            <a href="../projectsPage.html">Phoebe Badcock</a>
```
Comment (MightyWizard) [inline ExperiencePage/index.html]: Once you publish this will need to change to /

File: ExperiencePage/index.html [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+				<li>
+					<a href="#pre-career">Pre Career</a>
+				</li>
+                <li>
+				   <a href=#intro-week">CS Intro Week</a>
+				</li>
+		    </ul>
+		</div>
+		<div id="nav-section">
+			<ul>
```
Comment (MightyWizard) [inline ExperiencePage/index.html]: This doesn't need to be in a ul and li you can just make it text or a link (a href)

File: ExperiencePage/index.html [hunk 4]
```diff
@@ -0,0 +1,270 @@
...
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Phoebe Badcock Experience</title>
+    <link rel="stylesheet" href="styles.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" media="screen and (min-height: 800px)" href="monitor.css">
+	<link rel="stylesheet" media="screen and (max-width: 450px)" href="mobile.css">
+</head>
+<body>
+  &lt;!&ndash; START NAV &ndash;&gt;
```
Comment (MightyWizard) [inline ExperiencePage/index.html]: This is weird

File: ProjectsPage/styles.css [hunk 5]
```diff
@@ -0,0 +1,434 @@
+:root {
```
Comment (MightyWizard) [inline ProjectsPage/styles.css]: This should be the same as in the main style sheet so this should not be needed

File: ProjectsPage/styles.css
Comment (MightyWizard) [inline ProjectsPage/styles.css:1]: This doesn't really look like it's needed anymore.
Consider deleting this file

File: colors.html
Comment (MightyWizard) [inline colors.html:1]: This page isn't needed delete

File: index.html [hunk 6]
```diff
@@ -1,12 +1,305 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
+<html lang="en"><head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Phoebe Badcock Portfolio</title>
-    <link rel="stylesheet" href="./styles.css">
+    <link rel="stylesheet" href="styles.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (MightyWizard) [inline index.html]: Download this style sheet so you have it locally and just load the local version.

File: index.html [hunk 7]
```diff
@@ -1,12 +1,305 @@
...
-	<!--Elements visible to users go here-->
-</body>
-</html>
\ No newline at end of file
+  <!-- START NAV -->
+    <nav id="nav">
+		<div id="name">
+			Phoebe Badcock
+		</div>
+		<div onclick="menuFunction()" id="nav-list">
```
Comment (MightyWizard) [inline index.html:18]: onclick could possibly be in the nav tag

File: index.html [hunk 8]
```diff
@@ -1,12 +1,305 @@
...
+				<li>
+				  <a href="#extras">Extras</a>
+				</li>
+		    </ul>
+		</div>
+		<div onclick="menuFunction()" id="nav-contact">
+			<a href="#contact">Contact</a>
+		</div>
+		<!-- "Hamburger menu" / "Bar icon" to toggle the navigation links -->
+  		<a id="menu-icon" onclick="menuFunction()">
```
Comment (MightyWizard) [inline index.html]: rename to 'nav' or 'burger' icon

File: index.html [hunk 9]
```diff
@@ -1,12 +1,305 @@
...
+    		<i class="fa fa-bars" id="burger"></i>
```
Comment (MightyWizard) [inline index.html]: are you using the classes 'fa' and 'fa-bars'?

File: index.html [hunk 10]
```diff
@@ -1,12 +1,305 @@
...
+    </nav>
+  <!-- END NAV -->
+  <!-- START HOME SECTION -->
+	<section id="home">
+		<div class="home-content">
+			<img src="/Images/headshot.png" alt="Head Shot" id="headshot">
+			<div class="about">
+				<div class="picture-text">
+					<img src="https://d33wubrfki0l68.cloudfront.net/6e15034b71981e7553ddb6ba5acdf9b64c152af1/2e27b/assets/site/location.svg" alt="Location Icon" id="location-icon" width="15" height="15">
+					<p>Milton Keynes</p>
```
Comment (MightyWizard) [inline index.html]: Download this image and have it locally loaded

File: index.html [hunk 11]
```diff
@@ -1,12 +1,305 @@
...
+		<!-- "Hamburger menu" / "Bar icon" to toggle the navigation links -->
+  		<a id="menu-icon" onclick="menuFunction()">
+    		<i class="fa fa-bars" id="burger"></i>
+  		</a>
+    </nav>
+  <!-- END NAV -->
+  <!-- START HOME SECTION -->
+	<section id="home">
+		<div class="home-content">
+			<img src="/Images/headshot.png" alt="Head Shot" id="headshot">
```
Comment (MightyWizard) [inline index.html]: alt text headshot one word

File: index.html [hunk 12]
```diff
@@ -1,12 +1,305 @@
...
+			<div class="about">
+				<div class="picture-text">
+					<img src="https://d33wubrfki0l68.cloudfront.net/6e15034b71981e7553ddb6ba5acdf9b64c152af1/2e27b/assets/site/location.svg" alt="Location Icon" id="location-icon" width="15" height="15">
+					<p>Milton Keynes</p>
+				</div>
+				<div>
+					<h1>Software Developer</h1>
+				</div>
+				<div>
+					<h2>I create products to solve real world problems, from ideas to development and deployment.</h2>
```
Comment (MightyWizard) [inline index.html:57]: Consider using a <br> instead of a new h2 here

File: index.html [hunk 13]
```diff
@@ -1,12 +1,305 @@
...
+				<div class="summary-box">
+					<div class="summary-description">
+						<h3>Credit Analyst</h3>
+						<h4>Complaints Process</h4>
+						<ul>
+							<li>Streamlined the review process, cutting time per review from over 2 hours to under
+								30 seconds.</li>
+							<li>Handled updates and documented changes to the process.</li>
+						</ul>
+						<!--<h4>ONS Modelling</h4>
```
Comment (MightyWizard) [inline index.html]: remove commented out code

File: index.html [hunk 14]
```diff
@@ -1,12 +1,305 @@
...
+							<li>Time management</li>
+						</ul>-->
+						<h4>Education</h4>
+						<ul>
+							<li>University of Warwick, Mathematics BSc - First Class</li>
+							<li>A Levels: A*A*AA</li>
+						</ul>
+					</div>
+					<!--<a href="" target="_blank">Download CV</a>-->
+					<a href="/Phoebe Badcock CV.pdf" download="">
```
Comment (MightyWizard) [inline index.html]: consider renaming this to not have spaces (Spaces are not liked for file downloads)

File: index.html [hunk 15]
```diff
@@ -1,12 +1,305 @@
...
+						<button class="btn">
+							<i class="fa fa-download"></i>
```
Comment (MightyWizard) [inline index.html]: do use use 'fa' and 'fa-download'

File: index.html [hunk 16]
```diff
@@ -1,12 +1,305 @@
...
+				</div>
+			</div>
+		<!--</div>-->
+    </section>
+  	<!-- END CONTACT SECTION -->
+  	<footer>
+		<div id="reference">Hosted using GitHub Pages</div>
+	</footer>
+
+<div id="sp-installed"></div></body></html>
```
Comment (MightyWizard) [inline index.html]: Whats this sp-installed?

File: myscripts.js [hunk 17]
```diff
@@ -0,0 +1,50 @@
+function showCopyButton() {
```
Comment (MightyWizard) [inline myscripts.js]: The show and hide functions should be wrapped in a single function to toggle the copy button.
Use an 
```if copyInfo.style.display == "block"```
to determine if show or hide

File: myscripts.js [hunk 18]
```diff
@@ -0,0 +1,50 @@
...
+  var copyText = document.getElementById("email-copy");
+
+   // Copy the text inside the text field
+  navigator.clipboard.writeText(copyText.textContent);
+
+  var copyInfo = document.getElementById("copy-info");
+  copyInfo.innerText = "Copied";
+}
+
+function bobDown(icon) {
```
Comment (MightyWizard) [inline myscripts.js]: These are not longer needed

File: projectsPage.html [hunk 19]
```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Phoebe Badcock Projects</title>
+    <link rel="stylesheet" href="styles.css">
+	<script src="myscripts.js"></script>
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (MightyWizard) [inline projectsPage.html:9]: As home page download a local version

File: projectsPage.html [hunk 20]
```diff
@@ -0,0 +1,215 @@
...
+<body id="projects-page">
+  <!-- START NAV -->
+    <nav id="nav">
+		<div id="name">
+			Phoebe Badcock
+		</div>
+		<div onclick="menuFunction()" id="nav-list">
+			<ul >
+				<li>
+				  <a href="./index.html#home">Home</a>
```
Comment (MightyWizard) [inline projectsPage.html]: When published these will probably need to be changes to /#...

File: projectsPage.html [hunk 21]
```diff
@@ -0,0 +1,215 @@
...
+								characters sharing the same shift, each succeeding character experienced a shift
+								of n+1, where n represents the shift of the preceding character.
+							</li>
+							<li>Lastly, to maintain
+								code efficiency, I implemented the reverse encryption using the same function by
+								modifying a single parameter.
+							</li>
+						</ul>
+					</div>
+					<a href="" target="_blank">View More on GitHub</a>
```
Comment (MightyWizard) [inline projectsPage.html]: Missing link

File: projectsPage.html [hunk 22]
```diff
@@ -0,0 +1,215 @@
...
+									sharpen my coding skills and deepen my understanding of algorithms, enabling
+									the enhancing of my code efficiency. </li>
+								<li>This ongoing commitment serves as a
+									continuous effort to refine my coding proficiency,
+									and I intend to extend this practice to any new language I undertake.</li>
+							</ul>
+					</div>
+					<a href="https://github.com/PhoebeBC/leetcodeTop150" target="_blank">View More on GitHub</a>
+				</div>
+				<!--<div class="summary-box">
```
Comment (MightyWizard) [inline projectsPage.html]: Remove commented code

File: projectsPage.html [hunk 23]
```diff
@@ -0,0 +1,215 @@
...
+    </section>
+  <!-- END Complete SECTION -->
+<!-- START Coming Soon SECTION -->
+	<section id="coming-soon">
+		<div class="title"><h2>Coming Soon</h2></div>
+		    <div class="summary">
+				<div class="summary-box">
+					<div class="summary-description">
+					<h3>Recipie Book App</h3>
+							<ul>
```
Comment (MightyWizard) [inline projectsPage.html:157]: tabbing

File: styles.css [hunk 24]
```diff
@@ -0,0 +1,642 @@
...
+/*
+Inter,sans-serif
+*/
+/*ELEMENTS*/
+body {
+    background-color: white;
+    font-family: /*Inter,sans-serif*/'Poppins', sans-serif;
+    margin: 0;
+}
+
```
Comment (MightyWizard) [inline styles.css:18]: create
```
h1, h2, h3, h4, h5 {
    font-family: 'Raleway', sans-serif;
}```

File: styles.css [hunk 25]
```diff
@@ -0,0 +1,642 @@
...
+h1{
+    font-family: 'Raleway', sans-serif;
+    font-weight: 700;
+    margin-bottom:5%;
+    margin-top:1%;
+}
+
+h2, h3, h4, h5{
+    font-family: 'Raleway', sans-serif;
```
Comment (MightyWizard) [inline styles.css]: as per above

File: styles.css
Comment (MightyWizard) [inline styles.css:1]: Overall, remove all commented out code

File: styles.css [hunk 26]
```diff
@@ -0,0 +1,642 @@
...
+
+
+/*Navigation bar*/
+
+nav {
+    display: flex; /*Keeps all items on singular line*/
+     /*Equally spaces out div elements in nave bar*/
+    justify-content: space-between;
+    position: fixed; /*Mean nav bar will always be visible even when scrolling*/
+    left: 0px;
```
Comment (MightyWizard) [inline styles.css]: Instead of left and right use 
```width: 100%;```

File: styles.css [hunk 27]
```diff
@@ -0,0 +1,642 @@
...
+    right: 0px;
+    z-index: 10;
+    /* padding-bottom: 5px; */ /*Keeps nave bar looking nice*/
+    border-radius: 0px 0px var(--curve) var(--curve);
```
Comment (MightyWizard) [inline styles.css]: When using 0 you don't need to include px

File: styles.css [hunk 28]
```diff
@@ -0,0 +1,642 @@
...
+    background-color: var(--tertiary);
+}
+nav #menu-icon{
```
Comment (MightyWizard) [inline styles.css]: missing new line before

File: styles.css [hunk 29]
```diff
@@ -0,0 +1,642 @@
...
+    padding: 0;
+    margin: 0;
+}
+
+nav ul li{
+   /*Removes bullet points*/
+  float: left; /*Keeps list horizontal*/
+}
+
+nav ul a, nav ul li, #name, #nav-contact, nav div a{
```
Comment (MightyWizard) [inline styles.css]: Ideally you shouldn't need to style both an a and a li within a nav just style one

File: styles.css [hunk 30]
```diff
@@ -0,0 +1,642 @@
...
+    font-size: 1.5rem;
+    padding: 0.6rem 0.4rem;
+    font-weight: 550;
+    color: var(--white);
+}
+
+nav ul a:hover, nav ul li:hover, nav div a:hover{
```
Comment (MightyWizard) [inline styles.css]: as above

File: styles.css [hunk 31]
```diff
@@ -0,0 +1,642 @@
...
+}
+
+/*HOME*/
+
+#home, #contact{
+    background-color: var(--primary);
+    color: var(--white);
+    padding-top: 0;
+}
+#home{
```
Comment (MightyWizard) [inline styles.css]: new line

File: styles.css [hunk 32]
```diff
@@ -0,0 +1,642 @@
...
+
+#home p{
+    font-size: 0.5rem;
+}
+
+#home h2{
+    font-size: 1.5rem;
+}
+
+
```
Comment (MightyWizard) [inline styles.css:141]: extra new line

File: styles.css [hunk 33]
```diff
@@ -0,0 +1,642 @@
...
+/*PROJECTS- and repeated in extras, current and coming soon sections*/
+
+#projects, #current, #extras, #coming-soon{
+     background-color: white;
+     color: var(--tertiary);
+}
+#projects {
+box-shadow: inset 0px 5px 5px 0px var(--primary-shadow);
```
Comment (MightyWizard) [inline styles.css]: tabbing and new line after

File: styles.css [hunk 34]
```diff
@@ -0,0 +1,642 @@
...
+}
+#projects h2 , #current h2, #extras h2, #coming-soon h2{
+    color: var(--tertiary);
```
Comment (MightyWizard) [inline styles.css]: Not neeed as you specify it above line 178

File: styles.css [hunk 35]
```diff
@@ -0,0 +1,642 @@
...
+    font-size: 2rem;
+}
+#projects h3, #current h3, #extras h3,  #coming-soon h3 {
+    color: var(--white);
```
Comment (MightyWizard) [inline styles.css]: not needed as is specified below line 202

File: styles.css [hunk 36]
```diff
@@ -0,0 +1,642 @@
...
+#experience, #complete{
+     background-color: var(--secondary);
+     color: var(--tertiary);
+}
+#experience, #complete {
+box-shadow: 0px -5px 5px 0px var(--secondary-shadow);
+}
+
+#experience h2, #complete h2{
+    color: var(--tertiary);
```
Comment (MightyWizard) [inline styles.css]: specified above line 261

File: styles.css [hunk 37]
```diff
@@ -0,0 +1,642 @@
...
+    font-size: 2rem;
+}
+
+#experience h3 , #complete h3 {
+    color: var(--tertiary);
```
Comment (MightyWizard) [inline styles.css]: specified above line 261

File: styles.css [hunk 38]
```diff
@@ -0,0 +1,642 @@
...
+    margin: auto;
+    text-align: center;
+}
+
+/*box div*/
+#experience .summary-box, #complete .summary-box{
+    color: var(--tertiary);
+    background-color: white;
+}
+#complete .summary-box a:visited {
```
Comment (MightyWizard) [inline styles.css:286]: newline

File: styles.css [hunk 39]
```diff
@@ -0,0 +1,642 @@
...
+    color: var(--tertiary);
+}
+#complete .summary-box a:hover {
```
Comment (MightyWizard) [inline styles.css:290]: newline

File: styles.css [hunk 40]
```diff
@@ -0,0 +1,642 @@
...
+/*Download CV*/
+.btn {
+  color: var(--white);
+  height: 50px;
+  font-size: 20px;
+  margin-bottom: 5%;
+  background-color: var(--secondary);
+  cursor: pointer;
+}
+.btn:hover {
```
Comment (MightyWizard) [inline styles.css:304]: newline

File: styles.css [hunk 41]
```diff
@@ -0,0 +1,642 @@
...
+    color: white;
+    box-shadow: 4px 4px 5px #d7ccd4;
+}
+/*CONTACT*/
+
+#contact h2 {
+    color: var(--white);
+    font-size: 2rem;
+}
+#contact{
```
Comment (MightyWizard) [inline styles.css]: newline

File: styles.css [hunk 42]
```diff
@@ -0,0 +1,642 @@
...
+    box-shadow: 0px -5px 5px 0px var(--primary-shadow);
+}
+#contact .home-content{
```
Comment (MightyWizard) [inline styles.css]: newline

File: styles.css [hunk 43]
```diff
@@ -0,0 +1,642 @@
...
+    /* margin: 0 auto auto auto; */
+    display: flex;
+    flex-direction: column;
+    max-height: 60%;
+    /* margin-bottom: 120px; */
+    /* flex: auto; */
+}
+#message{
```
Comment (MightyWizard) [inline styles.css]: newline

File: styles.css [hunk 44]
```diff
@@ -0,0 +1,642 @@
...
+    font-size: 2rem;
+    color: var(--highlight);
+    font-family: 'Raleway', sans-serif;
```
Comment (MightyWizard) [inline styles.css]: font-family not needed

File: styles.css [hunk 45]
```diff
@@ -0,0 +1,642 @@
...
+    font-weight: 600;
+    margin: 2%;
+    text-align: center;
+    /* max-height: 33%; */
+}
+#message span{font-size: 3rem;}
```
Comment (MightyWizard) [inline styles.css]: newline tabbing

File: styles.css [hunk 46]
```diff
@@ -0,0 +1,642 @@
...
+    align-items: center;
+    justify-content: space-evenly;
+    width: 100%;
+    max-height: 75%;
+    /* height: 50%; */
+    /* max-height: -webkit-fill-available; */
+    /* flex-direction: row; */
+    /* flex-grow: 0; */
+}
+.contact-icon img {
```
Comment (MightyWizard) [inline styles.css]: newline

File: styles.css [hunk 47]
```diff
@@ -0,0 +1,642 @@
...
+    max-height: 100%;
+    max-width: 100%;
+}
+
+.contact-icon a, #content .summary div {
+    display: block;
+    max-height: 100%;
+    display: block;
+}
+.contact-icon {
```
Comment (MightyWizard) [inline styles.css]: newline

File: styles.css [hunk 48]
```diff
@@ -0,0 +1,642 @@
...
+}
+.contact-icon img {
+    max-height: 100%;
+    max-width: 100%;
+}
+
+.contact-icon a, #content .summary div {
+    display: block;
+    max-height: 100%;
+    display: block;
```
Comment (MightyWizard) [inline styles.css]: repeated

File: styles.css [hunk 49]
```diff
@@ -0,0 +1,642 @@
...
+}
+.contact-icon {
+    max-height: 50%;
+    max-width: 33%;
+    height: -webkit-fill-available;
+}
+#contact .picture-text{
```
Comment (MightyWizard) [inline styles.css]: Class no longer exists

File: styles.css [hunk 50]
```diff
@@ -0,0 +1,642 @@
...
+    width: min-content;
+    display: block;
+    max-height: inherit;
+}
+
+div#github-icon, div#linkedin-icon {
+    max-width: 125px;
+}
+#github-icon:hover, #linkedin-icon:hover {
```
Comment (MightyWizard) [inline styles.css]: newline

File: styles.css [hunk 51]
```diff
@@ -0,0 +1,642 @@
...
+    padding-top: 1%
+}
+#email-icon{
```
Comment (MightyWizard) [inline styles.css]: newline

File: styles.css [hunk 52]
```diff
@@ -0,0 +1,642 @@
...
+    /* height: 130px; */
+    /* width: 200px; */
+    /* object-fit: cover; */
+    padding-bottom: 2%;
+    margin: 0;
```
Comment (MightyWizard) [inline styles.css]: Should not need this margin 0

File: styles.css [hunk 53]
```diff
@@ -0,0 +1,642 @@
...
+}
+#email {
```
Comment (MightyWizard) [inline styles.css:386]: newline

File: styles.css [hunk 54]
```diff
@@ -0,0 +1,642 @@
...
+    cursor: pointer;
+    flex-direction: row;
+}
+.logo-text{
```
Comment (MightyWizard) [inline styles.css]: newline consider renaming as only used for email text

File: styles.css [hunk 55]
```diff
@@ -0,0 +1,642 @@
...
+    padding-top: 0;
```
Comment (MightyWizard) [inline styles.css]: dont need to say padding 0

File: styles.css [hunk 56]
```diff
@@ -0,0 +1,642 @@
...
+    margin-top: 0;
+    color: white;
+    /* font-size: 1.5rem; */
+    font-weight: 600;
+    text-align: center;
+}
+#email-link:hover{
+    #copy-button{
+        display: block;
```
Comment (MightyWizard) [inline styles.css]: change this to 
```
#email:hover #copy-info{
    display: block;
}
```
Then the JS show/hideCopyButton is not needed

File: myscripts.js (hunk 17 above)
Comment (MightyWizard) [reply myscripts.js]: As per line 400 of the css these function are not really needed at all

File: styles.css [hunk 57]
```diff
@@ -0,0 +1,642 @@
...
+        color: white;
+        z-index: 10;
+    }
+}
+#copy-info{
```
Comment (MightyWizard) [inline styles.css]: newline

Comment (MightyWizard) [reply styles.css]: color white is not needed and neither is font-weight

File: styles.css [hunk 58]
```diff
@@ -0,0 +1,642 @@
...
+}
+
+
+/*Footer*/
+
+footer {
+    margin-top: -20px;
+    display: flex; /*Keeps all items on singular line*/
+    justify-content: end; /*Equally spaces out div elements in nave bar*/
+    border-radius: var(--curve) var(--curve) 0px 0px;
```
Comment (MightyWizard) [inline styles.css]: remove px when it's 0

File: styles.css [hunk 59]
```diff
@@ -0,0 +1,642 @@
...
+    background-color: var(--tertiary);
+    color: white;
+}
+
+#reference {
+   font-size: 0.5rem;
+   padding: 0.5rem;
+   font-weight: 550;
+   color: var(--white);
```
Comment (MightyWizard) [inline styles.css]: color is not already specified above

File: styles.css [hunk 60]
```diff
@@ -0,0 +1,642 @@
...
+    margin: 0 5% 5%;
+    justify-content: space-between;
+    }
+  #contact .title{
+    margin-top: 5%;
+    margin-bottom: 0;
+  }
+}
+
+@media screen and (max-width: 500px) {
```
Comment (MightyWizard) [inline styles.css:501]: sort out tabbing in this block

File: styles.css [hunk 61]
```diff
@@ -0,0 +1,642 @@
...
+    position: absolute;
+    width: 10%;
+    align-items: center;
+    justify-content: center;
+    right: 0;
+    padding: 0 1%;
+    height: 45px;
+    }
+  #burger {
+    font-size: xx-large;
```
Comment (MightyWizard) [inline styles.css]: actually set the font size not this

File: styles.css [hunk 62]
```diff
@@ -0,0 +1,642 @@
...
+    color: var(--tertiary);
+    }
+  #home, #contact {
+    max-height: 100vh;
```
Comment (MightyWizard) [inline styles.css]: shouldn't need to set both height and max height

File: styles.css [hunk 63]
```diff
@@ -0,0 +1,642 @@
...
+    height: 100vh;
+  }
+  #projects{
+    max-height: 1400px;
```
Comment (MightyWizard) [inline styles.css]: This could cause problems on smaller screens

File: styles.css [hunk 64]
```diff
@@ -0,0 +1,642 @@
...
+    }
+  #message {
+    font-size: 5vw;
+    margin: auto 2%;
+  }
+  #message span {
+    font-size: 6vw;
+  }
+}
+/***********************************************************************************************************/
```
Comment (MightyWizard) [inline styles.css]: Whats this

**Review by MightyWizard — CHANGES_REQUESTED** [review]
Overall need to fix indenting and ensure only required code is included

File: styles.css (hunk 63 above)
Comment (PhoebeBC) [reply styles.css]: Removed as no longer needed

**Pull request merged**